arch/x86/include/ia32/kernel_arch_data.h

```diff
@@ -52,8 +52,6 @@
 
 #ifndef _ASMLANGUAGE
 
-#include <sys/util.h>
```

Comment:
Are we getting <sys/util.h> pulled in from another header?  don't see the duplicate here.